tests/Unit/Models/GroupTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Tests\Unit\Models;
+
+use App\Models\Bookmark;
+use App\Models\Group;
+use App\Models\Group\Member;
+use App\Models\Group\Workspace;
+use App\Models\LegiScan\Bill;
+use App\Models\User;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+
+class GroupTest extends TestCase
+{
+    /** @test */
+    public function groupsHaveMembers()
+    {
+        $group = Group::factory()->create();
+        $user1 = User::factory()->create();
```

Comment:
Don't use numbered variables like this. Why is there two users? For what? Is one supposed to be found and the other not in a query? Are they not friends? Are they husband and wife? Give them names like `$userToFind` and `$userNotToFind` or `$activeSubscription` and `$pausedSubscription` or `$brownCat` and `$greenCat`. That way someone else has a chance of understanding the logic they're being presented with in the test.